end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -252,63 +280,71 @@ time axis, while MS data is organized by spectrum, orthogonal to the retention
 time axis. The `chromatogram()` function facilitates the extraction of
 intensities along the retention time. However, access to chromatographic
 information is currently not as efficient and seamless as it is for spectral
-information. Work is underway to develop a `Chromatogram` object that is as
-comprehensive and user-friendly as the existing `Spectra` object. 
+information. Work is underway to develop/improve the infrastructure for
+chromatographic data through a new `Chromatograms` object aimed to be as
+flexible and user-friendly as the `Spectra` object.
+
```

Comment:
For now I would keep it as it is. We can then have a "version 2" of the paper later - that's actually possible in F1000 which I find also useful, because then it's possible to update a workflow - keeps it a bit more dynamic.